csvqb/Pipfile

```diff
@@ -6,6 +6,8 @@ verify_ssl = true
 [dev-packages]
 csvwlib-devtools = {editable = true,path = "./../devtools"}
 ipykernel = "*"
+pytest = "*"
```

Comment:
both of these should be installed by having `csvwlib-devtools` installed.

Reply:
Which two?

Reply:
pytest and black.